### PR TITLE
Fixing params in "rise-cache-not-running" event

### PR DESCRIPTION
**Pull request by stulees**
- Multiple params need to be provided in an object
- Revised unit test
- Minor patch version bump

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: rise-storage.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
 </dom-module>
 
 <!-- build:version -->
-<script>var sheetVersion = "1.4.0";</script>
+<script>var sheetVersion = "1.4.1";</script>
```
Comment (donnapep) [inline rise-storage.html]: @stulees Looks like this is stale.

**Comment by stulees** [comment]
@donnapep Nice catch, corrected. Been meaning to change this repo to work like how I setup financial component. Looks like I have my improvement for the next sprint ;)

**Comment by donnapep** [comment]
👍

**Pull request merged**